cg/meta/archive/archive.py

```diff
@@ -84,8 +99,116 @@ def archive_all_non_archived_spring_files(
 
         for archive_location in ArchiveLocations:
             self.archive_archive_location(
-                files_and_samples=files_and_samples, archive_location=archive_location
+                files_and_samples=files_and_samples,
+                archive_location=archive_location,
+            )
+
+    def retrieve_flowcell(self, flowcell_name: str) -> None:
```

Comment:
If we want to integrate the PDC archival into this flow, it sounded on @karlnyr as if it would be necessary. Maybe we can get some input?

Reply:
Regarding PDC we will likely want to rewrite the logic to also deal with single SPRING files instead of flow cells when we add that feature. The idea is to have it as modular as possible and not have to fetch unnecessary data

Reply:
I thought that PDC does not support to many requests coming their way and that that was the reason for fetching a whole flow cell?

Reply:
Luckily it seems like one can send multiple files per request (similar to DDN Dataflow).

Reply:
Removed